app/utils/theme.js

```diff
@@ -27,3 +29,18 @@ export function setNavigatorStyles(navigator, theme) {
         screenBackgroundColor: theme.centerChannelBg,
     });
 }
+
+export function getAllowedThemes(state) {
+    const allowedThemeKeys = getConfig(state).AllowedThemes.split(',');
```

Comment:
what if `AllowedThemes` is not set in the server config, shouldn't all themes be allowed?

Reply:
I'll change this to show all if AllowedThemes is empty